packaging/docker/Dockerfile.base

```diff
@@ -0,0 +1,29 @@
+# syntax=docker/dockerfile:1.0.0-experimental
```

Comment:
As we now have three different `Dockerfile`s, it would be nice to have a small comment at the top of this file, documenting 1) what the build will be tagged as, and 2) a short description of what it does.

Then the `Dockerfile`s should be understandable without having to also read the pipeline definition.

Reply:
Only one file now. :-)